### ISSUE TITLE
cannot find the required ev_run() function despite trying to link with -lev

### ISSUE THREAD

**Issue by salahmak** [Raymo111/i3lock-color#187]
## This issue is a....

<pre>
[x] Bug
[ ] Other kind of issue (Please describe in detail)
</pre>

## Current Behavior

<detail>
getting this error when installing:

```
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force 
autoreconf: configure.ac: tracing
autoreconf: configure.ac: not using Libtool
autoreconf: running: /usr/bin/autoconf --force
autoreconf: running: /usr/bin/autoheader --force
autoreconf: running: automake --add-missing --copy --force-missing
autoreconf: Leaving directory `.'
mkdir: created directory 'build/'
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking target system type... x86_64-pc-linux-gnu
checking for gsed... sed
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether make supports nested variables... (cached) yes
checking whether to enable maintainer-specific portions of Makefiles... yes
checking for gawk... (cached) gawk
checking for GNU make... make
checking for a sed that does not truncate output... /usr/bin/sed
checking whether to build with code coverage support... no
checking whether to enable debugging... yes
checking whether make supports the include directive... yes (GNU style)
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking dependency style of gcc... gcc3
checking for gcc option to accept ISO C99... none needed
checking how to run the C preprocessor... gcc -E
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking minix/config.h usability... no
checking minix/config.h presence... no
checking for minix/config.h... no
checking whether it is safe to define __EXTENSIONS__... yes
checking for stdbool.h that conforms to C99... yes
checking for _Bool... yes
checking for mode_t... yes
checking for off_t... yes
checking for pid_t... yes
checking for size_t... yes
checking for ssize_t... yes
checking for pid_t... (cached) yes
checking vfork.h usability... no
checking vfork.h presence... no
checking for vfork.h... no
checking for fork... yes
checking for vfork... yes
checking for working fork... yes
checking for working vfork... (cached) yes
checking whether lstat correctly handles trailing slash... yes
checking for working strnlen... yes
checking for atexit... yes
checking for dup2... yes
checking for ftruncate... yes
checking for getcwd... yes
checking for gettimeofday... yes
checking for localtime_r... yes
checking for memchr... yes
checking for memset... yes
checking for mkdir... yes
checking for rmdir... yes
checking for setlocale... yes
checking for socket... yes
checking for strcasecmp... yes
checking for strchr... yes
checking for strdup... yes
checking for strerror... yes
checking for strncasecmp... yes
checking for strndup... yes
checking for strrchr... yes
checking for strspn... yes
checking for strstr... yes
checking for strtol... yes
checking for strtoul... yes
checking for explicit_bzero... yes
checking for library containing floor... -lm
checking for library containing ev_run... no
configure: error: in `/home/salah/i3lock-color/build':
configure: error: cannot find the required ev_run() function despite trying to link with -lev
See `config.log' for more details
make: *** No targets specified and no makefile found.  Stop.
make: *** No rule to make target 'install'.  Stop.
```
</detail>

## Expected Behavior

installing without issues
## Reproduction Instructions

`$ chmod +x ./install-i3lock-color.sh`
`$ ./install-i3lock-color.sh`
## Environment
Output of `i3lock --version`:
<pre>
i3lock version: i3lock: version 2.11.1 © 2010 Michael Stapelberg.
</pre>

OS: Debian 10.6.
Desktop env: Xfce.
Where'd you get i3lock-color from?

<pre>
[ ] AUR package (which one?)
[x] Built from source yourself
[ ] Other (Please describe in detail)
</pre>

## Content of config.log:
https://pastebin.com/haB5BAC0

**Comment by Raymo111** [comment]
I just tried it myself on a laptop running Debian 9.15 with XFCE installed. I installed dependencies and cloned the latest version, then did exactly your repro instructions, and the install worked fine. I'm unable to repro as such. Did you install all the dependencies? I found that the command for Ubuntu
```
sudo apt install autoconf gcc make pkg-config libpam0g-dev libcairo2-dev libfontconfig1-dev libxcb-composite0-dev libev-dev libx11-xcb-dev libxcb-xkb-dev libxcb-xinerama0-dev libxcb-randr0-dev libxcb-image0-dev libxcb-util-dev libxcb-xrm-dev libxkbcommon-dev libxkbcommon-x11-dev libjpeg-dev
```
wouldn't work because `libxcb-util-dev` is `livxcb-util0-dev` on Debian. Replacing that worked. I've just added Debian dependency instructions.
